client/src/components/homeView/HomeShows.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import { Button } from '@material-ui/core'
+import { ArrowForward, ArrowBack } from '@material-ui/icons';
+import styles from './homeshows.module';
+import axios from 'axios';
+
+export default function HomeShows({query}) {
+  const [shows, setShows] = useState([]);
+  const [start, setStart] = useState(0);
+  const [end, setEnd] = useState(4);
+  
+  useEffect(() => {
+    axios.get(`/api/shows?query=${query}`)
```

Comment:
What is this query parameter? Why is it called 'query?" I think it's the sorting order, so wouldn't 'sort' be a better name for it?